packages/controller-utils/src/util.test.ts

```diff
@@ -441,6 +441,8 @@ describe('util', () => {
         const ethQuery = {
           getBlockByHash: (blockId: any, cb: any) => cb(null, { id: blockId }),
         };
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```

Comment:
Instead of `@ts-ignore`, I'd suggest using `@ts-expect-error`. That will actually result in an error if the below code does _not_ have any TypeScript errors, so it can catch possible regressions or other bugs.

Then you can also remove the ESLint comment.

Reply:
Oops, good point! Wrote this on auto-pilot

Reply:
Addressed in https://github.com/MetaMask/core/pull/1266/commits/d68106894261257ea2b5cefece7de2c6ffbfad6f